app/javascript/OutdoorMap/router.js

```diff
@@ -0,0 +1,8 @@
+Router = L.Routing.OSRMv1.extend({
```

Comment:
This file is also duplicated and should be deleted

Reply:
fixed it in 4860ac08b0adf6acb7cde1ddef0a22d1b10063d5